metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/query/request/SearchQueryBuilder.java

```diff
@@ -107,9 +116,11 @@
       fields.add(searchFieldConfig);
 
       if (SearchFieldConfig.detectSubFieldType(fieldSpec).hasDelimitedSubfield()) {
-        fields.add(SearchFieldConfig.detectSubFieldType(searchFieldConfig.getFieldName() + ".delimited",
-                searchFieldConfig.getBoost() * partialConfiguration.getFactor(),
-                fieldSpec.getSearchableAnnotation().getFieldType()));
+        final SearchableAnnotation searchableAnnotation = fieldSpec.getSearchableAnnotation();
+
+        fields.add(SearchFieldConfig.detectSubFieldType(searchFieldConfig.fieldName() + ".delimited",
+                searchFieldConfig.boost() * partialConfiguration.getFactor(),
+                searchableAnnotation.getFieldType(), searchableAnnotation.isQueryByDefault()));
```

Comment:
I think I prefer the method call here rather than passing true just because it does always evaluate to true due to the continue check. If we dismiss these alerts do they come back?